docs/gno-tooling/gno-tooling.md

```diff
@@ -4,5 +4,27 @@ id: gno-tooling
 
 # Gno Tooling
 
-Welcome to the **Gno Tooling** section for Gno. This section outlines programs & tools
-that are commonly used when developing applications with Gno.
\ No newline at end of file
+Welcome to the **Gno Tooling** section for Gno. This section outlines programs & tools that are commonly used when developing applications with Gno.
+
+## Gno Command Syntax Guide
```

Comment:
We should probably drop this from the section, I think we'll cover it more extensively in another place

Reply:
How about we keep this section for now?? (until more extensively pages comes in)